loadclient/randbytes_log_lines.go

```diff
@@ -0,0 +1,17 @@
+package loadclient
+
+import (
+	"math/rand"
+)
+
+const (
+	letterBytes          = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+)
+
+func getRandBytesLogLine(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Intn(len(letterBytes))]
+	}
+	return string(b)
+}
```

Comment:
@eranra do  you think renaming randbytes->synthetic make sense?

Reply:
done